pennylane/measurements/state.py

```diff
@@ -122,17 +122,16 @@ def circuit():
         with a compatible device.
     """
     wires = Wires(wires)
-    return StateMP(wires=wires)
+    return DensityMatrixMP(wires=wires)
 
 
 class StateMP(StateMeasurement):
```

Comment:
Should we forbid `wires` on the non-density matrix `StateMP`  for the moment? Then add a dedicated story in for `StateMP` can accept wires.